### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by anujdube12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Website - [fayas.me](https://fayas.me)
 
-Personal website of Fayas Noushad
+This ia a Personal website repository of Fayas Noushad, where you can get academic background , projects and contact info.
```
Comment (FayasNoushad) [inline README.md:3]: `This ia a` typo

Comment (FayasNoushad) [inline README.md:3]: Change to `This is my`

Comment (FayasNoushad) [inline README.md:3]: Remove whitespace from after `background` before the comma.

Comment (FayasNoushad) [inline README.md:3]: Please change `info` to `information`

Comment (anujdube12) [reply README.md]: I had changed it

Comment (anujdube12) [reply README.md]: updated correctly

**Pull request merged**